client/src/modules/depots/modals/depot.modal.html

```diff
@@ -129,8 +129,7 @@
         FORM.LABELS.MIN_MONTHS_SECURITY_STOCK
       </label>
       <input name="min_months_security_stock" class="form-control"
-        ng-model="DepotModalCtrl.depot.min_months_security_stock" type="number" min="1" autocomplete="off" required
-        bh-integer>
+        ng-model="DepotModalCtrl.depot.min_months_security_stock" type="number" min="0" autocomplete="off" required>
```

Comment:
We should consider making min="0.00001" (or similar) to prevent acceptance of zero.